<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeftMap</class>
 <widget class="QWidget" name="LeftMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QTableWidget" name="tableView">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>650</width>
       <height>120</height>
      </size>
     </property>
     <property name="gridStyle">
      <enum>Qt::SolidLine</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>70</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>25</number>
     </attribute>
     <column>
      <property name="text">
       <string>标签ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>X（m）</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Y（m）</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>θ（°）</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>inf.X</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>inf.Y</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>uwb.X</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>uwb.Y</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>路标ID</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGraphicsView" name="mapView"/>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>120</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>红外相机</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="infBox">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="infConnect">
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>连接</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>UWB标签</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="uwbBox_2">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="uwbConnect">
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>连接</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="startPro">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>开始</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stopPro">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>结束</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>infBox</tabstop>
  <tabstop>infConnect</tabstop>
  <tabstop>uwbBox_2</tabstop>
  <tabstop>uwbConnect</tabstop>
  <tabstop>startPro</tabstop>
  <tabstop>stopPro</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>mapView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
